syntax = "proto3";

package common;

// 提供默认返回值
message DefaultResponse {
    string msg = 1; // 仅为`success`
}
